readme.md

```diff
@@ -50,37 +49,65 @@ Adds filetype glyphs (icons) to other plugins such as [NERDTree], [vim-airline],
 <a name="quick-setup"></a>
 ## Quick Setup (TL;DR)
 
-1. Install the plugin per your usual method
-   _[(» More details... «)](#installation)_
-2. Download and install a patched font (or patch your own):
+1. Download and install a patched font (or patch your own):
   * [nerd-fonts]
    _[(» More details... «)](#font-installation)_
-3. Set font  _[(» More details... «)](#font-configuration)_
+2. Set font  _[(» More details... «)](#font-configuration)_
```

Comment:
Fix broken link "(» More details... «)". I suggest linking to heading I suggested in other comment ('Set Font')